### PR TITLE
Add JSON Schema example

### PR DESCRIPTION
**Pull request by danielgtaylor**
The example is pretty basic, but shows how a simple example could look and includes a list of strings.

### CODE REVIEW

**Comment by kylef** [comment]
Perhaps it would be nice to also include an example of an MSON described data structure with a JSON schema to emphasise you can override generated schemas for MSON structures.

File: examples/13. Named Endpoints.md [hunk 1]
```diff
@@ -6,6 +6,7 @@ This API example demonstrates how to define a standalone endpoint with an identi
 ## API Blueprint
 + [Previous: Advanced Action](12.%20Advanced%20Action.md)
 + [This: Raw API Blueprint](https://raw.github.com/apiaryio/api-blueprint/master/examples/13.%20Named%20Endpoints.md)
++ [Next: JSON Schema](14.%20JSON%20Schema.md)
```
Comment (pksunkara) [inline examples/13. Named Endpoints.md]: I don't know if JSON Schema has higher priority and should be listed a bit above in the examples directory. But that would mean editing a lot of examples. @zdne What is your directive on this? Should we just add the newer examples at the bottom?

File: examples/14. JSON Schema.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+                      "items": {
+                          "type": "string"
+                      }
+                  }
+                }
+            }
+
++ Response 204
+
+## Delete a note [DELETE]
```
Comment (pksunkara) [inline examples/14. JSON Schema.md]: Do we need this action?

**Comment by pksunkara** [comment]
> Perhaps it would be nice to also include an example of an MSON described data structure with a JSON schema to emphasise you can override generated schemas for MSON structures.

I think that should be another example on it's own along with the other Attributes examples.

Reviewed.

File: examples/14. JSON Schema.md (hunk 2 above)
Comment (danielgtaylor) [inline examples/14. JSON Schema.md]: No, it was just added to round out the example. I will remove it.

**Comment by danielgtaylor** [comment]
@kylef @pksunkara should I add another example with MSON or should that be a separate PR?

**Comment by pksunkara** [comment]
@danielgtaylor You can add it to this PR.

File: examples/14. JSON Schema.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+                "tags": [
+                    "todo",
+                    "work"
+                ]
+            }
+
+    + Schema
+
+            {
+                "type": "object",
```
Comment (kylef) [inline examples/14. JSON Schema.md]: Would additional unknown properties send in the request be allowed or rejected by the server?

Perhaps we should set `additionalProperties` in the schema to `false` to indicate that other unknown keys are not permitted in the request body.

Comment (kylef) [inline examples/14. JSON Schema.md]: Per [JSON Schema Validation 5.4.4 additionalProperties, properties and patternProperties](http://json-schema.org/latest/json-schema-validation.html#anchor64).

File: examples/14. JSON Schema.md [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+                "properties": {
+                  "title": {
+                      "type": "string"
```
Comment (kylef) [inline examples/14. JSON Schema.md]: We're mixing 2-spaces and 4-spaces for indentation in these JSON structures. Can we align the indentation to be consistent across all JSON bodies.

Above `type` and `properties` is indented by 4 spaces, then `title` is only indented by `two` spaces and then `type` inside `title` is indented by another 4 spaces.

File: examples/13. Named Endpoints.md (hunk 1 above)
Comment (zdne) [inline examples/13. Named Endpoints.md]: @pksunkara I think in this case it is OK if this goes last (as an advanced topic).

**Comment by pksunkara** [comment]
Just minor indentation issues.

**Comment by danielgtaylor** [comment]
@kylef @pksunkara updated with your feedback. Please take another look!

**Pull request merged**